client/src/js/modules/imaging/views/queuecontainer.js

```diff
@@ -586,6 +587,56 @@ define(['marionette',
             })            
         },
 
+        unqueueSelectedSamples: function(e) {
+            e.preventDefault()
+
+            var self = this
+            this.$el.addClass('loading');
+            var sids = _.map(this.qsubsamples.where({ isGridSelected: true }), function(ss) {return ss.get('BLSUBSAMPLEID')})
+
+            Backbone.ajax({
+                url: app.apiurl+'/sample/sub/queue',
+                data: {
+                    BLSUBSAMPLEID: sids,
+                    UNQUEUE: 1,
+                },
+                success: function(resp) {
+                    _.each(resp, function (r) {
+                        var ss = self.qsubsamples.findWhere({ BLSUBSAMPLEID: r.BLSUBSAMPLEID })
+                        ss.set({ READYFORQUEUE: '0' })
+                    })
+                },
+                complete: function(resp, status) {
+                    self.$el.removeClass('loading')
+                    self.refreshQSubSamples(self)
+                }
+            })
+        },
+
+        unqueueAllSamples: function(e) {
+            e.preventDefault()
+
+            var self = this
+            this.$el.addClass('loading');
+            Backbone.ajax({
+                url: app.apiurl+'/sample/sub/queue/cid/'+this.model.get('CONTAINERID'),
+                data: {
+                    queued: 1,
+                    UNQUEUE: 1,
+                },
```

Comment:
There is a body being sent with what is presumably a GET request

Reply:
It would be sent as URL parameters but would still be more suited to a post.